website/source/docs/providers/aws/r/elasticache_security_group.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: "aws"
+page_title: "AWS: aws_subnet"
+sidebar_current: "docs-aws-resource-elasticache-security-group"
+description: |-
+  Provides an VPC subnet resource.
+---
+
+# aws\_elasticache\_security\_<wbr>group
+
+Provides an ElastiCache Security Group to control access to one or more cache 
+clusters. Cache security groups are only used when you are creating a cache 
+cluster outside of a VPC.
```

Comment:
Might want to call this out a bit more. Maybe a Note: box or just bold the sentence and add the phrase "EC2 Classic Only" somewhere?